src/main/java/Bundle.properties

```diff
@@ -2069,3 +2069,32 @@ admin.api.migrateHDL.failure.must.be.hdl.dataset=Dataset was not registered as a
 admin.api.migrateHDL.success=Dataset migrate HDL registration complete. Dataset re-registered successfully.
 admin.api.migrateHDL.failure=Failed to migrate Dataset Handle id: {0}
 admin.api.migrateHDL.failureWithException=Failed to migrate Dataset Handle id: {0} Unexpected exception: {1}
+
+#Roles
```

Comment:
These are actually permissions; let's rename them that way.

Reply:
I'm not convinced we want to translate these - though yes to their descriptions. (it's probably ok to translate, it's just these are "reserved words" in the code.

Reply:
I will discuss with @amberleahey and @mhvezina to clarify whether those "reserved words" from line number 2074 to 2086 to be translated.

Reply:
Since those strings are displayed in the user interface, I would like to have a way to be able to translate them.

Reply:
I too agree we should translate these: they may be role types that are stored but they are readable/meaningful in the user interface.  This is working as per the updated pull request
![customroles](https://user-images.githubusercontent.com/17624834/47119366-9daa8600-d238-11e8-98ad-3302401d8502.png)